www/src/pages/TR/2025.10/index.html

```diff
@@ -219,46 +221,43 @@ <h1 id="title" class="title">Design Tokens Technical Reports 2025.10</h1>
     <a class="u-url url p-name fn" href="https://matthewstrom.com">Matthew Ström-Awn</a>
   </dd><dd class="editor p-author h-card vcard">
     <a class="u-url url p-name fn" href="https://www.donnavitan.com">Donna Vitan</a>
-  </dd><dd class="editor p-author h-card vcard">
-    <a class="u-url url p-name fn" href="https://x.com/mikekamminga">Mike Kamminga</a>
   </dd>
-
+      
       <dt>Authors:</dt><dd class="editor p-author h-card vcard">
     <a class="u-url url p-name fn" href="https://x.com/dbanksDesign">Daniel Banks</a>
   </dd><dd class="editor p-author h-card vcard">
-    <a class="u-url url p-name fn" href="https://x.com/kevinmpowell">Kevin Powell</a>
+    <a class="u-url url p-name fn" href="https://github.com/esthercheran">Esther Cheran</a>
   </dd><dd class="editor p-author h-card vcard">
-    <a class="u-url url p-name fn" href="https://x.com/andrewlhomme">Andrew L’Homme</a>
```

Comment:
I think these changes are correct; we made a fix in the preview version but not 2025.10